README.md

```diff
@@ -1,5 +1,8 @@
 # A List of Post-mortems!
 
+> This list of post-mortems has been included in the Verica Open Incident
+> Database (https://www.thevoid.community/) if you wish to browse them there.
```

Comment:
We have a list of other postmortems at the bottom of this document. Please move this comment to there.